### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by beanumber**
I hope I didn't mess anything up here...all this should do is:
- add the rendered confidence interval in HTML
- fix a typo from "the the" to "the"

I didn't look carefully at the rendered HTML. Can those be removed from the repo?

### CODE REVIEW

**Comment by mine-cetinkaya-rundel** [comment]
This PR just pinged me on email, not sure why since it's from a year ago. Do you know @beanumber?

**Comment by beanumber** [comment]
The PR is from tonight. Some of the commits are really old, but they have been superseded by the changes you all have made recently. I fetched the upstream changes and merged before I committed. If you look at the files that have actually changed in this PR, it's just a tiny change to one of the Rmd and all the other changes are to the rendered HTML.

**Comment by mine-cetinkaya-rundel** [comment]
Thanks! I saw the Sep 8 2015 date and was confused.

You also asked about removing the rendered HTML files. I'm usually in favor of that, although in this case having them here means we can link to them from openintro.org, and hence if any updates are made files are updated automatically. What was your reasoning for wanting to remove HTML files? Does this purpose outweigh it in your opinion?

**Comment by beanumber** [comment]
No big deal either way. The pros of removing the HTML is that they are derivative, so they aren't strictly necessary. Also, even though they are text files they tend to be large and include lots of crap that you can't read, consequently diffs don't work as well. I figured they were here so that they could be automatically synced with the website, but I wonder whether that could be done in a different location.

Basically, if I change something in one of the Rmd's, then should also have to render and commit the changes to the HTML?

**Comment by mine-cetinkaya-rundel** [comment]
Yup, that's currently how we're doing it. I agree that it's an additional step -- though I'd never change an Rmd and not render it (that's how I check the change didn't introduce a bug). Let's stick to this for now as I'm not sure how else to handle syncing of the HTMLs linked from the website, but I'll think about it.
